src/python/qeqhipster/utils.py

```diff
@@ -54,7 +55,16 @@ def replace_identity_gates_with_rx(circuit):
     return circuit
 
 
-def replace_XX_YY_ZZ_gates_with_decomposition(circuit):
+def replace_iswap_gates_with_decomposition(circuit):
+    for gate in circuit.gates:
+        if gate.name == "ISWAP":
+            raise NotImplementedError(
+                "ISWAP gate is currently not supported for qHipster integration."
+            )
+    return circuit
+
+
+def replace_pauli_rotation_gates_with_decomposition(circuit):
```

Comment:
Is `pauli_rotation_gates` the right name for these gates? To me, this makes me think of `Rx`, `Ry`, and `Rz`, but I might be wrong...

Reply:
That's how they called in qulacs, that's why I used that .
`two_qubit_pauli_rotation_gates` ?

Reply:
See the newest commit.